specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json

```diff
@@ -0,0 +1,1918 @@
+            "get": {
+                "tags": [
+                    "Connections"
+                ],
+                "summary": "Get a connection",
+                "operationId": "Connections_Get",
+                "description": "Get a specific connection",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/subscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/resourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/connectionNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An API connection",
+                        "schema": {
+                            "$ref": "#/definitions/ApiConnectionDefinition"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "Connections"
+                ],
+                "summary": "Replace an existing connection",
+                "operationId": "Connections_CreateOrUpdate",
+                "description": "Creates or updates a connection",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/subscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/resourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/connectionNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiConnectionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "New API connection created",
+                        "schema": {
+                            "$ref": "#/definitions/ApiConnectionDefinition"
+                        }
+                    },
+                    "201": {
+                        "description": "Already existing API connection updated",
+                        "schema": {
+                            "$ref": "#/definitions/ApiConnectionDefinition"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "tags": [
+                    "Connections"
+                ],
+                "summary": "Update an existing connection",
+                "operationId": "Connections_Update",
+                "description": "Update a connection",
```

Comment:
Verb is with present tense below, i.e. Deletes
Should this here be Updates then?